install/prerequisites.adoc

```diff
@@ -729,16 +729,45 @@ It depends on the target host of the deployment and uses the computed value of
 |===
 | *9200*
 |TCP
-|For Elasticsearch API use. Required to be internally open on any infrastructure
+| For Elasticsearch API use. Required to be internally open on any infrastructure
 nodes so Kibana is able to retrieve logs for display. It can be externally
 opened for direct access to Elasticsearch by means of a route. The route can be
 created using `oc expose`.
 
 | *9300*
 |TCP
-|For Elasticsearch inter-cluster use. Required to be internally open on any
+| For Elasticsearch inter-cluster use. Required to be internally open on any
 infrastructure node so the members of the Elasticsearch cluster might communicate
 with each other.
+
+| *9090*
+| TCP 
+| For Prometheus web console use.
+
+| *9100*
+| TCP
+| For the Prometheus Node-Exporter, which exports hardware and operating system metrics.
+
+| *8443*
+| TCP
+| For node hosts to communicate to the master API, for the node hosts to
+post back status, to receive tasks, and so on.
+
+| *10250*
```

Comment:
This port needs to be allowed from masters (already explained above) and infra nodes to any master and node. Although it is said above that it is explicitly needed from masters, it would be good to also explain that for the metrics to work, the needed source is the infra nodes.